include/scheduler/pending_process_callbacks.hpp

```diff
@@ -11,7 +12,7 @@ namespace edge {
 // TLDR: singleton vs everything owned by scheduler, food for thought
 class PendingProcessCallbacks {
 public:
-    static constexpr uint8_t MAX_READY_CALLBACKS = 10;
+    static constexpr uint8_t MAX_READY_CALLBACKS = 20;
```

Comment:
should we bump this to like 64? I can see a world where a lot could reasonably build up in one quantum

Reply:
agree here but not really the actual issue